### PR TITLE
unity to lts of 2019

### PR DESCRIPTION
**Pull request by dnharjani-sa**

### CODE REVIEW

**Review by strangeweekend — APPROVED** [review]
🌮

File: scripts/0.download.sources.sh [hunk 1]
```diff
@@ -4,30 +4,49 @@ rm -rf build
...
+
+# TODO implement tag and make it override branch
+
+
+############################################################
 # step 1: download android source code
 ############################################################
 
-# rm -rf source/android/*
+if [[ $platform == 'android' || $platform == 'all' ]]
```
Comment (GuyMcLeanSA) [inline scripts/0.download.sources.sh:27]: Should we default "platform" == "all" in the case of there being no platform arg. I might have missed something but it looks like no platform arg results in no build.....or does unity build just with what it has already without cleaning?

Comment (GuyMcLeanSA) [reply scripts/0.download.sources.sh:27]: Ignore me, just saw it in build.sh

**Review by GuyMcLeanSA — APPROVED** [review]
🌮

File: scripts/3.build.unity.sh [hunk 2]
```diff
@@ -14,51 +21,55 @@ mkdir build/unity/$source/demo/Assets/Plugins/iOS
...
+if [[ $platform == 'ios' || $platform == 'all' ]]
+then
+  cp -r build/ios/* build/unity/$source/demo/Assets/Plugins/iOS/
+fi
+############################################################
 # step 4: build as UnityPackage
 ############################################################
 
-/Applications/Unity/Hub/Editor/2019.3.10f1/Unity.app/Contents/MacOS/Unity \
+/Applications/Unity/Hub/Editor/2019.4.28f1/Unity.app/Contents/MacOS/Unity \
```
[the remark this answers is not in the document]
Comment (dnharjani-sa) [reply scripts/3.build.unity.sh:67]: Yeah probably worth doing in the future - for now i'll just update it to the LTS release of Unity

**Pull request merged**